config/linter.go

```diff
@@ -47,6 +47,10 @@ type SpectralLinter struct {
 	// Rules are a list of Spectral ruleset file locations
 	Rules []string `json:"rules"`
 
+	// Script identifies the path to the spectral script to use for linting.
+	// If not defined linting will look for spectral-cli on $PATH.
+	Script string
```

Comment:
For consistency (other config keys are camelCase),

```suggestion
	Script string `json:"script"`
```